README.md

```diff
@@ -40,14 +40,9 @@ $ make install-hooks
 ```
 
 ### Environment Variables
-There is a file called `.envrc` that contains environment variables that are used by components that should behave differently in different environments.
+Various scripts and commands rely on environment variables being set. These are documented with the commands.
 
-If you have [direnv](https://direnv.net/) installed, your shell will automatically pick up these environment variables.
```

Comment:
Why is .envrc being removed? Local values would be useful. At the very least leaving an example file would be useful.